scripts/pt/portugal_geoedif.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os.path
+
+# The source data can be obtained by running
+# esri2geojson https://inenetw05.ine.pt:6443/arcgis/rest/services/GeoEdif/GEOEDIF_AC25/MapServer/0 pt-azores25.json
+# esri2geojson https://inenetw05.ine.pt:6443/arcgis/rest/services/GeoEdif/GEOEDIF_AC26/MapServer/0 pt-azores26.json
+# esri2geojson https://inenetw05.ine.pt:6443/arcgis/rest/services/GeoEdif/GEOEDIF_MAD/MapServer/0 pt-madeira.json
+# esri2geojson https://inenetw05.ine.pt:6443/arcgis/rest/services/GeoEdif/GEOEDIF_CONT/MapServer/0 pt-continente.json
+# In practice the continental dataset is too large (>3M points), so the server fails before returning the count. Forcing esri2geojson to use OIDs does the trick
```

Comment:
Do you want the raw jsons uploaded too for inspection?
I figured that since this is a large download (taxes their server substantially) and that this particular source is unlikely to get updated (a version of data for new census is likely to be published as a different layer), the processing is fine.